### DISCUSSION TITLE
imagemagick in an apache module

### DISCUSSION THREAD

**Discussion by riccardovacirca** [ImageMagick/ImageMagick#4907] [Development]
Is it possible to use the imagemagick c API in the context of an apache module? Can the MagickWandGenesis and MagickWandTerminus methods cause any problems?

**Comment by urban-warrior** [comment]
Yes of course.  We recommend you build ImageMagick with modules enabled (`--with-modules`) and use either the Magick++ (C++) or MagickWand API's (C) for optimal performance.    We are not aware of any issues with using ImageMagick in an Apache module. 

ImageMagick recommended practices strongly encourage you to configure a [security policy](https://imagemagick.org/script/security-policy.php) that suits your local environment.
